docs/api.md

```diff
@@ -558,3 +559,70 @@ coll = kinto.collection("articles", {
 There's currently no way to deal with adding transformers to an already filled remote database; that would mean remote data migrations, and both Kinto and Kinto.js don't provide this feature just yet.
 
 **As a rule of thumb, you should only start using transformers on an empty remote collection.**
+
+## Id schemas
```

Comment:
nit: use "ID", as id is used to refer to [something completely different](https://en.wikipedia.org/wiki/Id,_ego_and_super-ego#Id).